packages/apps/admin/src/routes/entities/components/Action/index.js

```diff
@@ -3,6 +3,7 @@ import {actions} from 'tocco-app-extensions'
 
 const actionMap = {
   'widget-code-copy': lazy(() => import(/* webpackChunkName: "actions" */ './actions/WidgetCodeCopy')),
+  score: lazy(() => import(/* webpackChunkName: "actions" */ './actions/ScoreBoard')),
```

Comment:
nitpick: I would keep the same name for the `./actions/ScoreBoard.js` file as the name of the action

In this case the filename would be`Score.js`